services/device-registry-base/src/main/java/org/eclipse/hono/service/management/credentials/X509CertificateCredential.java

```diff
@@ -148,20 +153,6 @@ private static X509Certificate deserialize(final byte[] base64EncodedX509Certifi
         }
     }
 
-    /**
-     * {@inheritDoc}
-     */
-    @Override
-    protected Predicate<String> getAuthIdValidator() {
-        return authId -> {
-            if (Strings.isNullOrEmpty(authId)) {
-                return false;
-            }
-            final X500Principal distinguishedName = new X500Principal(authId);
-            return distinguishedName.getName(X500Principal.RFC2253).equals(authId);
-        };
-    }
```

Comment:
we should keep the validator but only check for the auth ID being not null nor empty.
The superclass constructor only checks for null ...